perception/autoware_multi_object_tracker/config/input_channels.param.yaml

```diff
@@ -3,80 +3,134 @@
+          can_trust_existence_probability: true
+          can_trust_extension: true
+          can_trust_classification: true
+          can_trust_orientation: true
         optional:
           name: "apollo"
           short_name: "Lap"
       lidar_apollo_validated:
         topic: "/perception/object_recognition/detection/apollo/validation/objects"
-        can_spawn_new_tracker: true
+        flags:
+          can_spawn_new_tracker: true
+          can_trust_existence_probability: true
+          can_trust_extension: true
+          can_trust_classification: true
+          can_trust_orientation: true
         optional:
           name: "apollo"
           short_name: "Lap"
       # LIDAR-CAMERA - DNN
       # cspell:ignore lidar_pointpainting pointpainting
       lidar_pointpainting:
         topic: "/perception/object_recognition/detection/pointpainting/objects"
-        can_spawn_new_tracker: true
+        flags:
+          can_spawn_new_tracker: true
+          can_trust_existence_probability: true
+          can_trust_extension: true
+          can_trust_classification: true
+          can_trust_orientation: true
         optional:
           name: "pointpainting"
           short_name: "Lpp"
       lidar_pointpainting_validated:
         topic: "/perception/object_recognition/detection/pointpainting/validation/objects"
-        can_spawn_new_tracker: true
+        flags:
+          can_spawn_new_tracker: true
+          can_trust_existence_probability: true
+          can_trust_extension: true
+          can_trust_classification: true
+          can_trust_orientation: true
         optional:
           name: "pointpainting"
           short_name: "Lpp"
       # CAMERA-LIDAR
       camera_lidar_fusion:
         topic: "/perception/object_recognition/detection/clustering/camera_lidar_fusion/objects"
-        can_spawn_new_tracker: true
+        flags:
+          can_spawn_new_tracker: true
+          can_trust_existence_probability: false
+          can_trust_extension: false
+          can_trust_classification: true
+          can_trust_orientation: false
         optional:
           name: "camera_lidar_fusion"
           short_name: "CLf"
       # CAMERA-LIDAR+TRACKER
       detection_by_tracker:
         topic: "/perception/object_recognition/detection/detection_by_tracker/objects"
-        can_spawn_new_tracker: false
+        flags:
+          can_spawn_new_tracker: false
+          can_trust_existence_probability: false
+          can_trust_extension: false
+          can_trust_classification: false
+          can_trust_orientation: false
         optional:
           name: "detection_by_tracker"
           short_name: "dbT"
       # RADAR
       radar:
         topic: "/sensing/radar/detected_objects"
-        can_spawn_new_tracker: true
+        flags:
+          can_spawn_new_tracker: true
+          can_trust_existence_probability: true
+          can_trust_extension: false
+          can_trust_classification: true
```

Comment:
[just a comment]
Do we need partial trust to this? (we may believe unknown/vehicle classification for radar object but not detailed one)